### DISCUSSION TITLE
Need help to package my library

### DISCUSSION THREAD

**Discussion by leosok** [micropython/micropython#11783] [Libraries & Drivers]
hi there!
I wrote a library I use to drive  a 8x32 WS2812b Matrix of Neopixels. It works well for me, and I want to share it with folks. I was thinking to package it for upip, but the information I found was not very helpful. 
Do you think it's worth the effort? Is the repository for upip and mip the same?
And if both answers are "yes", could someone please help me to package this:
https://github.com/leosok/micropython_neopixel_matrix

**Comment by jimmo** [comment]
Please see https://docs.micropython.org/en/latest/reference/packages.html

**Comment by jposada202020** [comment]
Hiya just went through the same process. The docs are good and explain everything, if you want to see an example in action take a look here https://github.com/jposada202020/MicroPython_KX132 :)
